### ISSUE TITLE
Support backticks for arguments in Sphinx style

### ISSUE THREAD

**Issue by kalibyrn** [terrencepreilly/darglint#170]
Given the following function:

```python
def key_if_required(key: str, required: bool = True) -> dict:
    """
    Check if key is required and return a dictionary.

    :param `key`: The name of the key to check
    :param `required`: Whether the key is required or not
    :returns: Dictionary
    """
    if key not in ["dogs", "cats"]:
        return {"default": key}
    return {"required": required}
```

darglint will report these errors when using Sphinx formatting:

```
test_me.py:key_if_required:4: DAR102: + `key`
test_me.py:key_if_required:4: DAR102: + `required`
test_me.py:key_if_required:4: DAR101: - key
test_me.py:key_if_required:4: DAR101: - required
```

The backticks are nice because they allow highlighting in editors. If there were an option to ignore backticks for arguments that would be great.

![image](https://user-images.githubusercontent.com/619806/123125466-89ba7780-d416-11eb-9008-d1e9f9178805.png)
